release-tools/Makefile.prog

```diff
@@ -26,6 +26,6 @@ install:${PROG}
 clean:
 	rm -f ${PROG} ${SRCS:.c=.o}
 
-${PROG}:${SRCS:.c=.o}
+${PROG}:${SRCS:.c=.o} ${PROJECT_LIBS}
 	${CC} -o ${PROG} ${SRCS:.c=.o} ${LDFLAGS} ${LDADD_EXTRA} ${LDADD_REQ} ${LDADD_POSIX}
```

Comment:
I think it would be better to put `${PROJECT_LIBS}` here (before `${LDFLAGS}`) rather than including it in `LDADD_REQ`.  It shouldn't matter but theoretically it might if someone has an `LDADD_EXTRA` which conflicts with a symbol we use.

Reply:
I had that a few hours ago, but then I discovered that it broke `make -f Makefile.BSD`.

Reply:
Oh, hmm.  That could be solved by adding it to `LDADD` in `Makefile.inc` I think?

Reply:
Huh, I forgot about that file.  If nothing else, that would be a better place to stuff `libspiped.a`.

I'll revise this PR with `liball.a` and `Makefile.inc`.